### ISSUE TITLE
Swipe the body to open and close bottomsheet?

### ISSUE THREAD

**Issue by 10000multiplier** [stipsan/react-spring-bottom-sheet#118]
Currently only swiping on the header or footer activates the opening/closing of the bottom sheet.

The UX would greatly improve if you could swipe the body too.

**Comment by mechaadi** [comment]
Yes, I am looking forward to this as well. Gonna be great for UX.

**Comment by stipsan** [comment]
Yes, an early version of this library actually supported it, but it had to be disabled because it failed to work consistently on Android and iOS. I plan on revisiting this when I'm done updating to the latest [`react-spring`](https://github.com/stipsan/react-spring-bottom-sheet/pull/117) and [`react-use-gesture`](https://github.com/stipsan/react-spring-bottom-sheet/pull/78) versions 😄

**Comment by bulby97** [comment]
hey @stipsan any updates on this ? Thank you :)

**Comment by stipsan** [comment]
Hello @bulby97, try giving the new [`expandOnContentDrag `](https://github.com/stipsan/react-spring-bottom-sheet#expandoncontentdrag) prop a shot :) 

It's enabled on this [demo](https://react-spring-bottom-sheet.cocody.dev/fixtures/scrollable).
